scipy/cluster/tests/test_vq.py

```diff
@@ -289,3 +289,24 @@ def test_kmeans_large_thres(self):
         res = kmeans(x, 1, thresh=1e16)
         assert_allclose(res[0], np.array([4.]))
         assert_allclose(res[1], 2.3999999999999999)
+        
+    def test_kmeans2_kpp_low_dim(self):
+        # Regression test for gh-11462
+        prev_res = np.array([[-1.95266667, 0.898],
+                             [-3.153375, 3.3945]])
+        np.random.seed(42)
+        res, _ = kmeans2(TESTDATA_2D, 2, minit='++')
+        assert_allclose(res, prev_res)
+
+    def test_kmeans2_kpp_high_dim(self):
+        # Regression test for gh-11462
+        n_dim = 100
+        size = 10
+        centers = np.vstack([5 * np.ones(n_dim),
+                             -5 * np.ones(n_dim)])
+        np.random.seed(42)
+        data = np.vstack([np.random.multivariate_normal(centers[0], np.eye(n_dim), size=size),
+                          np.random.multivariate_normal(centers[1], np.eye(n_dim), size=size)])
+        # 20x100 dataset
```

Comment:
Shouldn't it be what follows ? 
```python
# 20x100 dataset
```
I think that you can get rid of this comment: this test is simple enough to directly get the shape of `data` here. :)

Reply:
It is 20 samples because each cluster is 10 normal distributed points.

But I removed the comment, I agree it is unnecessary.

I appreciate your feedback.